docs/appActions.md

```diff
@@ -1023,6 +1023,18 @@ Notifies autoplay has been blocked
 
 
 
+### savePassword() 
```

Comment:
Would these be better named as `passwordSaved` / `passwordUpdated`? (to be more consistent with others that you introduced, like urlBarTextChanged)